hil_testing/Magma_Automations/scripts/base.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2020 The Magma Authors.
```

Comment:
There are a number of new files where the year in the licence header is `2020` instead of `2022`, can we fix that?

Reply:
Done; I realized halfway through that I had the wrong year... I corrected forward but not backwards :P .. thanks for checking.